tests/saitests/sai_qos_tests.py

```diff
@@ -731,6 +731,13 @@ def runTest(self):
             packet_length = int(self.test_params['packet_size'])
         else:
             packet_length = 64
+
+        is_dualtor = bool(self.test_params.get('is_dualtor', False))
+        is_t0 = bool(self.test_params.get('is_t0', False))
+        def_vlan_mac = self.test_params.get('def_vlan_mac', None)
+        if is_dualtor and is_t0 and bool(def_vlan_mac):
```

Comment:
instead of bool(def_vlan_mac), you can use the check  'def_vlan_mac is not None'

Reply:
bool can cover empty string also

Reply:
solved